core/tests/protractor_desktop/explorationTranslationTab.js

```diff
@@ -257,7 +257,7 @@ describe('Exploration translation and voiceover tab', function() {
       await expEditorTranslationTab.expectCorrectStatusColor(
         'final card', RED_STATE_PROGRESS_COLOR);
       await expEditorTranslationTab.expectNumericalStatusAccessibilityToMatch(
-        '1 item translated out of 8 items');
+        '1 item translated out of 9 items');
```

Comment:
I had to increase the count as we now need to translate the interaction text as well.